tools/generate_docker_image_names.py

```diff
@@ -0,0 +1,128 @@
+"""
+Dockerリポジトリ名、バージョン文字列、カンマ区切りのプレフィックスを受け取り、
+バージョン文字列付きのDockerイメージ名を改行区切りで標準出力に出力する
+
+例1
+$ python3 ./tools/generate_docker_image_names.py \
+  --repository "voicevox/voicevox_engine" \
+  --version "0.22.0" \
+  --prefix ",cpu,cpu-ubuntu22.04"
+voicevox/voicevox_engine:0.22.0
+voicevox/voicevox_engine:cpu-0.22.0
+voicevox/voicevox_engine:cpu-ubuntu22.04-0.22.0
+
+例2
+$ python3 ./tools/generate_docker_image_names.py \
+  --repository "voicevox/voicevox_engine" \
+  --version "latest" \
+  --prefix "nvidia,nvidia-ubuntu22.04"
+voicevox/voicevox_engine:nvidia-latest
+voicevox/voicevox_engine:nvidia-ubuntu22.04-latest
+
+例3
+$ python3 ./tools/generate_docker_image_names.py \
+  --repository "voicevox/voicevox_engine" \
+  --version "latest" \
+  --prefix ""
+voicevox/voicevox_engine:latest
```

Comment:
ここの例良いですね！
まあ完全に好みな気がしますが、がっつり簡略化しても全然通じる＆わかりやすくなるかも。

```suggestion
例1
$ python3 ./tools/generate_docker_image_names.py \
  --repository "REPO" \
  --version "VER" \
  --prefix ",A,B"
REPO:VER
REPO:A-VER
REPO:B-VER

例2
$ python3 ./tools/generate_docker_image_names.py \
  --repository "REPO" \
  --version "latest" \
  --prefix ""
REPO:latest
```

うーん･･･。まあこれは好みですね！！！　そのままでも良さそう！！

Reply:
簡略化していいと思いました！

例2もVERにしてコミットしてみます。

```suggestion
例1
$ python3 ./tools/generate_docker_image_names.py \
  --repository "REPO" \
  --version "VER" \
  --prefix ",A,B"
REPO:VER
REPO:A-VER
REPO:B-VER

例2
$ python3 ./tools/generate_docker_image_names.py \
  --repository "REPO" \
  --version "VER" \
  --prefix ""
REPO:VER
```

Reply:
コミットしてから、latestがバージョン文字列だとは気づきにくいというのを踏まえると、あえて例2はlatestのままでもよかった気もちょっとしてきましたが、まあVERでもいいかなと思っています。